masterlist.yaml

```diff
@@ -6363,6 +6364,21 @@ plugins:
       - NPC.Race
       - Relev
 
+  # BNW-Override-NVR3-Faces
+  - name: 'BNW-Override-NVR3-Faces-Patch.esm'
+  - name: 'BNW-Override-NVR3-Faces-Patch-place.esp'
+    tag:
+      - Actors.Voice
+      - NPC.Eyes
+      - NPC.FaceGen
+      - NPC.Hair
+      - NPC.Race
+    clean:
+      - crc: 0xFAEE2D5F
+        util: 'FNVEdit v4.1.5d'
+      - crc: 0x03F47422
+        util: 'FNVEdit v4.1.5d'
+
```

Comment:
Should the tags apply to both patches?

```yaml
  # BNW-Override-NVR3-Faces
  - name: 'BNW-Override-NVR3-Faces-Patch.esm'
  - name: 'BNW-Override-NVR3-Faces-Patch-place.esp'
    tag:
      - Actors.Voice
      - NPC.Eyes
      - NPC.FaceGen
      - NPC.Hair
      - NPC.Race
```

```yaml
  # BNW-Override-NVR3-Faces
  - name: 'BNW-Override-NVR3-Faces-Patch(-place)?\.es[mp]'
    tag:
      - Actors.Voice
      - NPC.Eyes
      - NPC.FaceGen
      - NPC.Hair
      - NPC.Race
```

And is the cleaning data for both patches?
Those should be applied to the actual file.
```yaml
    clean:
      - crc: 0xFAEE2D5F
        util: 'FNVEdit v4.1.5d'
      - crc: 0x03F47422
        util: 'FNVEdit v4.1.5d'
```

```yaml
  - name: 'BNW-Override-NVR3-Faces-Patch.esm'
    clean:
      - crc: 0xFAEE2D5F
        util: 'FNVEdit v4.1.5d'
  - name: 'BNW-Override-NVR3-Faces-Patch-place.esp'
    clean:
      - crc: 0x03F47422
        util: 'FNVEdit v4.1.5d'
```

Reply:
Both Override patches affect NPC records.
One of them is an esm, the other a standard non-esm flagged esp.
So I would say the tags apply.

That is cleaning data for the two patches indeed. My mistake.